src/AgConfigure.cpp

```diff
@@ -53,22 +75,17 @@ String Configuration::getLedBarModeName(LedBarMode mode) {
  *
  */
 void Configuration::saveConfig(void) {
```

Comment:
Propose to name the file according to the class name, so AgConfigure -> Configuration